modules/stanford_soe_helper_collection_views/stanford_soe_helper_collection_views.views_default.inc

```diff
@@ -10,6 +10,356 @@
+  $handler->display->display_options['pager']['options']['items_per_page'] = '1';
+  $handler->display->display_options['pager']['options']['offset'] = '0';
+  $handler->display->display_options['style_plugin'] = 'default';
+  $handler->display->display_options['row_plugin'] = 'fields';
+  /* Relationship: Entity Reference: Referencing entity */
+  $handler->display->display_options['relationships']['reverse_field_s_mag_article_collection_node']['id'] = 'reverse_field_s_mag_article_collection_node';
+  $handler->display->display_options['relationships']['reverse_field_s_mag_article_collection_node']['table'] = 'node';
+  $handler->display->display_options['relationships']['reverse_field_s_mag_article_collection_node']['field'] = 'reverse_field_s_mag_article_collection_node';
+  /* Field: Content: Intro label */
+  $handler->display->display_options['fields']['field_s_collectlon_intro_label']['id'] = 'field_s_collectlon_intro_label';
+  $handler->display->display_options['fields']['field_s_collectlon_intro_label']['table'] = 'field_data_field_s_collectlon_intro_label';
+  $handler->display->display_options['fields']['field_s_collectlon_intro_label']['field'] = 'field_s_collectlon_intro_label';
+  $handler->display->display_options['fields']['field_s_collectlon_intro_label']['label'] = '';
+  $handler->display->display_options['fields']['field_s_collectlon_intro_label']['element_label_colon'] = FALSE;
+  /* Field: Content: Title */
+  $handler->display->display_options['fields']['title']['id'] = 'title';
+  $handler->display->display_options['fields']['title']['table'] = 'node';
+  $handler->display->display_options['fields']['title']['field'] = 'title';
+  $handler->display->display_options['fields']['title']['label'] = '';
+  $handler->display->display_options['fields']['title']['alter']['word_boundary'] = FALSE;
+  $handler->display->display_options['fields']['title']['alter']['ellipsis'] = FALSE;
+  $handler->display->display_options['fields']['title']['element_label_colon'] = FALSE;
+  $handler->display->display_options['fields']['title']['link_to_node'] = FALSE;
+  /* Field: Content: Subtitle */
+  $handler->display->display_options['fields']['field_s_collection_subtitle']['id'] = 'field_s_collection_subtitle';
+  $handler->display->display_options['fields']['field_s_collection_subtitle']['table'] = 'field_data_field_s_collection_subtitle';
+  $handler->display->display_options['fields']['field_s_collection_subtitle']['field'] = 'field_s_collection_subtitle';
+  $handler->display->display_options['fields']['field_s_collection_subtitle']['label'] = '';
+  $handler->display->display_options['fields']['field_s_collection_subtitle']['element_label_colon'] = FALSE;
+  /* Sort criterion: Content: Post date */
+  $handler->display->display_options['sorts']['created']['id'] = 'created';
+  $handler->display->display_options['sorts']['created']['table'] = 'node';
+  $handler->display->display_options['sorts']['created']['field'] = 'created';
```

Comment:
@minorwm 
Should this sort by the article publishing date instead? I think this is sorting by the collection creation date.